### PR TITLE
Update set-netfirewallprofile.md

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to Log allowed, Log ignored and Log Blocked

### CODE REVIEW

**Comment by e0i** [comment]
@JohanFreelancer9 Minor copy editing is needed for this PR. Thanks.

**Comment by e0i** [comment]
@get-itips @tseward

Peer review needed for this PR. Thanks

File: docset/windows/netsecurity/set-netfirewallprofile.md [hunk 1]
```diff
@@ -488,9 +488,9 @@ Accept wildcard characters: False
 Specifies how to log the allowed packets in the location specified by the *LogFileName* parameter.
 The acceptable values for this parameter are: False, True, or NotConfigured. 
 
-- True: Windows writes an entry to the log whenever an incoming or outgoing connection is prevented by the policy. 
-- False: No logging for dropped connections.
-This parameter removes the setting from the GPO, which results in the policy not changing the value on the computer when the policy is applied. 
+- True: Windows writes an entry to the log whenever an incoming or outgoing connection is Allowed by the policy.
```
Comment (JohanFreelancer9) [inline docset/windows/netsecurity/set-netfirewallprofile.md]:
```suggestion
- True: Windows writes an entry to the log whenever an incoming or outgoing connection is allowed by the policy. 
```

**Comment by get-itips** [comment]
Looks ok

**Comment by tseward** [comment]
Looks good.

**Comment by e0i** [comment]
@AndreaBarr The PR can be merged post-migration.

**Pull request merged**